rke/k8s_service_options.go

```diff
@@ -85,6 +85,14 @@ func loadK8sVersionServiceOptions() map[string]v3.KubernetesServicesOptions {
 			Kubeproxy:      getKubeProxyOptions(),
 			Scheduler:      getSchedulerOptions(),
 		},
+		"v1.16.11-rancher1-1": {
```

Comment:
I think this PR can be closed except we need to add this section in.

Reply:
Currently, there is no `v1.16.11-rancher1-1` version in `dev-v2.5`, the latest we have is `v1.16.10-rancher2-2`